### PR TITLE
Update to latest bioconda-recipes infrastructure

### PR DESCRIPTION
**Pull request by nsoranzo**

### CODE REVIEW

**Comment by bgruening** [comment]
@nsoranzo any particular reason to do this now? I'm working on the bulk branch to remove the r-channel. After this is merged it would be a perfect time to update it.

**Comment by nsoranzo** [comment]
Because PRs won't work with the updated bioconda-utils, see https://travis-ci.org/galaxyproject/conda-iuc/builds/240396349

**Comment by bgruening** [comment]
Oh we have a PR :)

File: .travis.yml [hunk 1]
```diff
@@ -11,13 +11,18 @@ services:
 
 install: scripts/travis-setup.sh
 
-script: travis_wait 119 scripts/travis-run.sh
+script: scripts/travis-run.sh
 
 env:
   global:
     - SUBDAGS=1
     - BIOCONDA_UTILS_TAG=master
```
Comment (bgruening) [inline .travis.yml]: maybe you want to pin here a specific commit

**Review by bgruening — APPROVED** [review]

File: .travis.yml (hunk 1 above)
Comment (nsoranzo) [reply .travis.yml]: The [latest](https://github.com/bioconda/bioconda-utils/commit/415709197fc3390aecbb4ab61e90950b5a192da5)?

File: .travis.yml [hunk 2]
```diff
@@ -16,7 +16,7 @@ script: scripts/travis-run.sh
 env:
   global:
     - SUBDAGS=1
-    - BIOCONDA_UTILS_TAG=master
+    - BIOCONDA_UTILS_TAG=415709197fc3390aecbb4ab61e90950b5a192da5
     - BIOCONDA_UTILS_BUILD_ARGS="--loglevel=info"
```
Comment (bgruening) [inline .travis.yml]: This one: https://github.com/bioconda/bioconda-utils/commit/7a67a25088d5243dce8f662b779ae54fd41dd67b

The update to latest conda did not worked :(

Comment (nsoranzo) [reply .travis.yml]: Done.

**Comment by nsoranzo** [comment]
I think the Travis build failure would be solved if we first apply the change to `config.yml`, but I don't think it's worthwhile to split this PR in 2. Merge?

**Comment by bgruening** [comment]
It's not :)

**Pull request merged**